jazzmin/templatetags/jazzmin.py

```diff
@@ -265,3 +265,15 @@ def header_class(header, forloop):
         classes.append("sorting")
 
     return ' '.join(classes)
+
+
+@register.simple_tag()
+def safe_classes(classes):
+    """
+    Remove reserved classes from CSS class list
+    """
+    if type(classes) == str:
+        classes = classes.split(' ')
+
+    excluded = ['collapse']
+    return ' '.join([x for x in classes if x not in excluded])
```

Comment:
Maybe, if it is just a string, you could use `replace` instead for readability. It depends whether you want to add more stuff to `excluded` in the future or not.. TBH, I would call that tag just `strip_collapse_class`, as that's what it is really doing. 🙂 

```
@register.simple_tag()
def strip_collapse_class(classes):
    """
    Remove collapse class from CSS class list
    """
    if type(classes) == str:
        classes = classes.replace('collapse', '')

    return classes
```

It looks like it can only be a string, so that would suffice. Anyway, cheese for thought! 🧀

Reply:
Good shout, you drove me to a much cleaner solution, https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#cut

Reply:
That's just brilliant